charts/argo-rollouts/Chart.yaml

```diff
@@ -19,4 +19,4 @@ annotations:
     url: https://argoproj.github.io/argo-helm/pgp_keys.asc
   artifacthub.io/changes: |
     - kind: fixed
-      description: Fixed rendering of plugins in the ConfigMap
+      description: Updated plugin values.yaml example and it's implementation to not need to include the stringification or the plugins block that it used to
```

Comment:
As a result, maybe `Type of trafficRouterPlugins and trafficRouterPlugins is changed from object to list` is enough ?